app/code/community/Meanbee/Footerjs/Helper/Data.php

```diff
@@ -7,6 +7,13 @@ class Meanbee_Footerjs_Helper_Data extends Mage_Core_Helper_Abstract {
     const REGEX_DOCUMENT_END  = '#</body>\s*</html>#isU';
 
     const XML_CONFIG_FOOTERJS_ENABLED = 'dev/js/meanbee_footer_js_enabled';
+    const XML_CONFIG_FOOTERJS_EXCLUDED_BLOCKS = 'dev/js/meanbee_footer_js_excluded_blocks';
+
+    const EXCLUDE_FLAG = 'data-meanbee-skip="true"';
+    const EXCLUDE_FLAG_PATTERN = 'data-meanbee-skip';
```

Comment:
I don't require "meanbee" to be in the attribute.  Can we use something that's a bit more descriptive/semantic over what it's doing, e.g data-footer-js-skip please?